.github/workflows/pypi_release.yml

```diff
@@ -0,0 +1,100 @@
+name: Build and Release Python Bindings to PyPI
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      tag:
+        type: string
+        default: ''
+        required: false
+        description: Ref to build
+      test_pypi:
+        type: boolean
+        description: 'Test release: publish on test.pypi.org'
+        default: false
+
+jobs:
+  build-sdist:
+    name: Build sdist
+    runs-on: ubuntu-latest
+    steps:
+      - name:   Checkout the repo
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.inputs.tag }}
+
+      - name: 🐍 Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+
+      - name:   Install build dependencies
+        run: |
+          python -m pip install build
+
+      - name: 📦 Build the sdist
+        working-directory: ./openvdb/openvdb/python
+        run: |
+          python -m build --sdist
+
+      - name:   Upload artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: dist
+          path: ./openvdb/openvdb/python/dist/
+
+  build-wheels:
```

Comment:
I would recommend to avoid building wheels at release time. It's better to build in PRs and when merging PRs and to release the artifacts generated by these builds at release time. The main reason being that lots of things can change between the time a PR is merged and a release is made on the GitHub Actions side. The OS images used change all the time.

Reply:
Okay, following your advice I've split the workflow into two: `wheels.yml` will build python bindings when a PR is made, and the second will release those wheels when a release is made.

Reply:
Great! I'll review this soon (in he upcoming day(s)).